sos/report/plugins/ceph_common.py

```diff
@@ -117,6 +114,87 @@ def setup(self):
             self.add_cmd_output([f"microceph {cmd}" for cmd in cmds],
                                 subdir='microceph')
 
+            dqlite_crt = "/var/snap/microceph/common/state/cluster.crt"
+            self.add_cmd_output(
+                f"openssl x509 -in {dqlite_crt} -noout -dates",
+                subdir="microceph"
+            )
+
+            db_path = "/var/snap/microceph/common/state/database"
+
+            # Check for inconsistent dqlite db intervals
+            self.add_dir_listing(
+                db_path,
+                suggest_filename="ls_microceph_dqlite_dir",
+                subdir="microceph",
+            )
+
+            self.add_copy_spec(
+                [
+                    f"{db_path}/info.yaml",
+                    f"{db_path}/cluster.yaml",
+                    f"{db_path}/../daemon.yaml",
+                ]
+            )
```

Comment:
Just to follow previous standards, lets use the following structure instead, so that the brackets are together here

```python
            self.add_copy_spec([
                f"{db_path}/info.yaml",
                f"{db_path}/cluster.yaml",
                f"{db_path}/../daemon.yaml",
            ])
```